Gemfile

```diff
@@ -90,12 +90,12 @@ gem 'bcrypt', '~> 3.1.6'
 # See: config/initializers/rabl_hack.rb
 gem 'rabl', '~> 0.13.0'
 gem 'multi_json', '~> 1.12.1'
-gem 'oj', '~> 3.0.5'
+gem 'oj', '~> 3.0.6'
```

Comment:
Have you looked into what the `optimize_rails` initializer does for oj? https://github.com/ohler55/oj/blob/master/pages/Rails.md Have only glimpsed over it, but it may be worth enabling if that isn't the case already.